_scripts/webpack.web.config.js

```diff
@@ -108,13 +109,14 @@ const config = {
     ]
   },
   node: {
-    __dirname: true,
+    __dirname: false,
     __filename: isDevMode,
   },
   plugins: [
     new webpack.DefinePlugin({
       'process.env.PRODUCT_NAME': JSON.stringify(productName),
-      'process.env.IS_ELECTRON': false
+      'process.env.IS_ELECTRON': false,
+      '__dirname': 'window.location.pathname.endsWith("/")?window.location.pathname.substring(0, window.location.pathname.length - 1):window.location.pathname'
```

Comment:
I would prefer that you create a helper function (it can go straight into helpers/utils.js instead of store/utils.js, that way I don't have to move it over in a later PR) for this, based on https://github.com/FreeTubeApp/FreeTube/blob/development/src/renderer/i18n/index.js#L57-L67. That makes the code much more maintainable and less cryptic, personally I wouldn't look at the `__dirname` variable and assume that it is being reassinged in a webpack config. It will also handle `index.html` being in the URL path, which this implementation doesn't at the moment.

The function could be named something along the lines of createWebURL(path).